components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -1184,6 +1184,15 @@ public static String getOAuth2DCREPUrl(String tenantDomain) throws URISyntaxExce
             return oauth2TokenEPUrl;
         }
 
+        public static String getOAuth2DCREPUrl() throws URISyntaxException {
+
+            String oauth2TokenEPUrl = OAuthServerConfiguration.getInstance().getOAuth2DCREPUrl();
+            if (StringUtils.isBlank(oauth2TokenEPUrl)) {
+                oauth2TokenEPUrl = IdentityUtil.getServerURL("/api/identity/oauth2/dcr/v1.0/register", true, false);
```

Comment:
It would be better to define a constant for "/api/identity/oauth2/dcr/v1.0/register"

Reply:
This is fixed after adding String constants for OAuth2 endpoints.